### ISSUE TITLE
Add refresh feature to sidebar

### ISSUE THREAD

**Issue by haivp3010** [sqlectron/sqlectron-gui#389]
There should be a way to reload the database list on the sidebar. Otherwise, we would need reload the whole app every time there are changes to databases from outside of the app

**Comment by BornaP** [comment]
There's support for refresh of every database (context menu on database name). However for refresh of a current connection you're supposed to reload the app for now. I don't think it's a big issue, but if you find it so, PR is welcome :)

**Comment by synle** [comment]
This could be a cool little project for me to get started on the code base.

Do you think we should HIDE ALL the EXPANDED tables and databases after we REFRESH the list?

**Comment by synle** [comment]
Looking at the layout, I think we might already had the refresh button on the top of the page. I am not sure if it makes sense to add another refresh button on the sidebar.


I have some basic code working for the refresh button on the side bar. If this is something you want, I can adjust the style accordingly.

![screen shot 2017-12-21 at 1 10 12 pm](https://user-images.githubusercontent.com/3792401/34275062-7590e5da-e650-11e7-8f85-5d295ef2abf3.png)

**Comment by maxcnunes** [comment]
Yeah, looks like the old **Reconnect** button on the top right already does that. @synle does that new **Refresh** button works differently than the **Reconnect** button? If it has the same behavior I rather keep a single button for it.
